src/packages/solid/components/Icon/Icon.tsx

```diff
@@ -37,19 +37,19 @@ export type IconProps = {
   /**
    * path to image or inline SVG XML
    */
-  icon?: string;
+  src?: string;
 };
 
 const Icon: Component<IconProps> = props => {
   return (
     <View
       {...props}
       // TODO, wait to see if fixed is needed
-      src={props.icon}
+      src={props.src}
       style={styles.container}
       color={hexColor(props.color) || styles.container?.color}
```

Comment:
We should discuss colors - end users should pass in 0xrgba values and not do the conversion here. Also if color is in styles.container you don't need a fallback. so color = can be removed